From 746324f55bf5df33aa096e4958f3ce25a58f0478 Mon Sep 17 00:00:00 2001
From: Ting Liu <ting.liu@freescale.com>
Date: Tue, 25 Feb 2014 21:10:02 -0600
Subject: [PATCH] manual/Makefile: add missing backslash

Fix the below error:
| /usr/bin/perl libm-err-tab.pl $pwd/../sysdeps >> libm-err-tmp
| Can't stat /../sysdeps: No such file or directory

Signed-off-by: Ting Liu <ting.liu@freescale.com>
---
 manual/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/manual/Makefile b/manual/Makefile
index ad19d2b..78bc18c 100644
--- a/manual/Makefile
+++ b/manual/Makefile
@@ -109,7 +109,7 @@ libm-err.texi: stamp-libm-err
 stamp-libm-err: libm-err-tab.pl $(wildcard $(foreach dir,$(sysdirs),\
 						     $(dir)/libm-test-ulps))
 	pwd=`pwd`; \
-	$(PERL) $< $$pwd/../ports > libm-err-tmp
+	$(PERL) $< $$pwd/../ports > libm-err-tmp; \
 	$(PERL) $< $$pwd/../sysdeps >> libm-err-tmp
 	$(move-if-change) libm-err-tmp libm-err.texi
 	touch $@
-- 
1.7.9.7

